### DISCUSSION TITLE
[Meetup] 42 spotkanie Microsoft Azure User Group Poland

### DISCUSSION THREAD

**Discussion by wi5nia** [AzureCommunityPL/group#59] [Community Content]
Cześć,
Wracamy po wakacjach z nową, odświeżoną formułą naszych meetupów. Spośród zgłoszonych tematów nasza społeczność wybrała cztery najciekawsze sesje. Szykuje się spora dawka wiedzy, nie może Was zabraknąć na spotkaniu.

Meetup:
- [https://www.meetup.com/Microsoft-Azure-Users-Group-Poland/events/272374642/](https://www.meetup.com/Microsoft-Azure-Users-Group-Poland/events/272374642/)

STREAM:
- [https://www.youtube.com/watch?v=PTgfO5L2d_0](https://www.youtube.com/watch?v=PTgfO5L2d_0)
- [https://www.facebook.com/azuredaypl/videos/624058874973149/](https://www.facebook.com/azuredaypl/videos/624058874973149/)

Agenda:
- 18:00-18:05, **Powitanie** - @mgrabarz 
- 18:05-18:35, **"Partycje w kolejkach"**, @aryll 
- 18:35-19:20, **"Automatyczne budowanie obrazu dla Azure WVD"**, @huczas
- 19:20-19:30, Przerwa
- 19:30-20:15, **"Godzilla - case study of system collecting data"**, @wolszakp
- 20:15-21:00, **"ARM on steroids czyli Azure Deployment Manager"**, @kamil-mrzyglod

#### @aryll  - "Od zawsze .NET, od pewnego czasu wywalanie Azure do góry nogami."
Partycje w kolejkach - "Krótka sesja o partycjach na Service Bus, Event Hub. Wady, zalety, różnice w podejściu."

#### @huczas - "Obecnie pracuję na stanowisku Azure Cloud Consultant w firmie BlueSoft. Pracuję w zespole Azure przy wdrożeniach do różnych klientów."

Automatyczne budowanie obrazu dla Azure WVD - "W trakcie pandemii mieliśmy kilka wdrożeń WVD, potencjalnie będzie takich więcej i być może komuś przyda się taki mały zalążek pomysłu w Azure DevOps na wdrożenie automatycznego budowania obrazów z przeznaczeniem na Azure Windows Virtual Desktop. W trakcie sesji chciałem pokazać w jaki sposób przygotowuje się obraz systemu Windows, instaluje na nim paczkę kilku aplikacji a następnie zamyka go."

#### @wolszakp - "Software engineer with over 10 years of experience in development, designing and architecting.
With hands on experience in Azure Cloud since 2017."

Godzilla - case study of system collecting data - "This will be a story about system to collect (big) data for later data analysis.
This will contain requirements, design decisions with stress put to question "why".
How incoming business needs where addressed using Azure platform and what is our experience after.
In the session there will be some technical aspects about: EventHub, AppService, DataLake Store, DataLake Analytics, Azure Function, AppInsights, StreamAnalytics Job, AzureData Factory, RedisCache, ServiceBus"

#### @kamil-mrzyglod - "Chmurowy ekspert z 10 latami doświadczenia w różnych projektach IT. Trener, Microsoft Azure MVP, od czasu do czasu prelegent i autor książek oraz publikacji nt. Microsoft Azure."

ARM on steroids czyli Azure Deployment Manager - "Typowe wdrożenie z użyciem Azure Resource Managera skupia się na pojedynczym regionie. Co jednak zrobić, kiedy musimy rozproszyć naszą infrastrukturę? Jak skoordynować proces wdrożenia? Podczas prezentacji spróbujemy wykorzystać funkcjonalność Azure Deployment Managera i ocenić, czy zdaje on egzamin."

**Comment by huczas** [comment]
Kontakt do mnie [hwit.pl](https://hwit.pl) a materiał który pokazywałem jest poniżej:
[Azure DevOps](https://dev.azure.com/hwitkowski-dev/WVD-images/)

**Comment by aryll** [comment]
Prezentacja o partycjach: https://www.slideshare.net/AgnieszkaCent/partycje-w-kolejkach-agnieszka-cent

[**Spotkanie Architektura pod CRUD 17/09 20:00**](https://github.com/AzureCommunityPL/group/discussions/48)

Kontakt do mnie:
https://www.linkedin.com/in/agnieszka-cent
https://github.com/aryll

**Comment by wolszakp** [comment]
Podrzucam prezentacje na temat Godzilli:
https://github.com/wolszakp/godzilla-public/blob/master/2020-09-09-Godzilla_case_study_AzureUserGroup.pptx

Kontakt do mnie:
https://www.linkedin.com/in/piotr-wolszakiewicz/
https://github.com/wolszakp

Link do feedback-u:
https://forms.gle/LAiRaMZhXaGdEAqd9

**Comment by kamil-mrzyglod** [comment]
Prezka: [ARM on steroids.pptx](https://github.com/AzureCommunityPL/group/files/5197543/ARM.on.steroids.pptx)

Social:
https://twitter.com/Kamil_Mrzyglod
https://www.linkedin.com/in/kamil-mrzyg%C5%82%C3%B3d-31470376/
kamil@thecloudtheory.com

I pamiętajcie aby podpinać kompa do ładowania!
